localstack/services/cloudwatch/alarm_scheduler.py

```diff
@@ -28,8 +29,7 @@
 STATE_OK = "OK"
 STATE_INSUFFICIENT_DATA = "INSUFFICIENT_DATA"
```

Comment:
These values are also in the generated ASF API: `localstack.aws.api.cloudwatch.StateValue`